### ISSUE TITLE
Ignoring a part of a document

### ISSUE THREAD

**Issue by elias-blm** [reuseman/flashcards-obsidian#16]
Hello, it would to be great to be able to ignoring a part of a document.
In my workflow, I have an Anki Deck named "Obsidian" where every cards I add from Obsidian end up. Once a day, I sort it and push the flashcards in the deck where they belong.
But, sometimes, i rewrite flashcards in the same document. The cards I already sorted are re-added in the Obsdian deck. So it would be great to *add a separator to ignore Flashcards before a certain indicator*.

Thanks!

**Comment by reuseman** [comment]
Hey, indeed it could be a good feature, but from a personal point of view I do not find a lot of value in it, yet. In these period I do not have a lot of time to implement new stuff, so I have to be picky.

Anyway, how do you think this should be done by using simple Markdown and Obsidian features? I do not want to break it and add too much custom syntax over it.

**Comment by elias-blm** [comment]
Hello,
I totally understand that you don't have time! I will try to pull request for this as soon as I can't!
I think the best way to do it is with a custom keyword specified in the settings. For example: *break point* which you're sure will not write by mistake.
I'm not a markdown expert, so maybe there's a simpler way to do it! Like with --- (but I believe it's already used)
Anyway, thanks for this plugin!

**Comment by reuseman** [comment]
Yeah @elias-blm, probably the only way to go is to have a customizable keyword, something like `cards-ignore`. Anyway, if you do the pull request, it will be more than welcomed!
